HW4/src/getMin.js

```diff
@@ -0,0 +1,4 @@
+function getMin(a, ...b) {
```

Comment:
could you use instead 'a' and 'b' an 'arguments' object ?

Reply:
+

Reply:
@AgentPrus , perfect comment